hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/helpers/BlockDeletingServiceMetrics.java

```diff
@@ -86,6 +108,34 @@ public void incrFailureCount() {
     this.failureCount.incr();
   }
 
+  public void incrReceiveTransactionCount(long count) {
+    receivedTransactionCount.incr(count);
+  }
+
+  public void incrReceiveRetryTransactionCount(long count) {
```

Comment:
Typo: it should be "incr`Received`RetryTransactionCount".

Reply:
done.